test/types/schema.test.ts

```diff
@@ -374,12 +374,12 @@ export function autoTypedSchema() {
     boolean2?: boolean;
     boolean3?: boolean;
     boolean4?: boolean;
-    mixed1?: any;
-    mixed2?: any;
-    mixed3?: any;
-    objectId1?: Schema.Types.ObjectId;
-    objectId2?: Schema.Types.ObjectId;
-    objectId3?: Schema.Types.ObjectId;
+    mixed1?: unknown;
```

Comment:
I think we should keep mixed inferred as any and not unknown, so I would like to keep it any. "mixed1, mixed2 and mixed3"